libs/api/domains/fishing-license/src/lib/fishing-license.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Inject, Injectable } from '@nestjs/common'
+import {
+  SkipApi,
+  UtgerdirApi,
+  FishingLicenseCodeType,
+} from '@island.is/clients/fishing-license'
+import { Auth, AuthMiddleware } from '@island.is/auth-nest-tools'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+
+@Injectable()
+export class FishingLicenseService {
+  constructor(
+    private readonly shipApi: SkipApi,
+    private readonly utgerdirApi: UtgerdirApi,
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+  ) {}
+
+  async getShips(nationalId: string, auth: Auth) {
```

Comment:
As this is a GQL service we always have a authenticated user, I would recommend to use the `User` interface instead of `Auth`. Same on the `getFishingLicense` below.

Reply:
Changed to User